gcc/rust/lex/rust-token.cc

```diff
@@ -172,6 +173,43 @@ Token::get_str () const
   return *str;
 }
 
+namespace {
+enum class Context
+{
+  String,
+  Char
+};
+
+const std::map<char, std::string> matches = {
+  {'\t', "\\t"}, {'\n', "\\n"},	 {'\r', "\\r"},
+  {'\0', "\\0"}, {'\\', "\\\\"}, {'\v', "\\v"},
+};
+
+std::string
+escape_special_chars (const std::string &source, Context ctx)
+{
+  std::stringstream ss;
```

Comment:
```suggestion
  std::stringstream stream;
```

very small nit, but I always hvae trouble understanding what `ss` refers to while `stream` is clearer IMO

Reply:
Fixed!